### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Patrick-Daly-AA**
Added link to innersourcing document

# Alaska Airlines Pull Request

Added a link to the innersourcing document for Alaska airlines employees, contractors, and vendors

**Fixes:** # (issue, if applicable)

## Summary:

Adding a link to the Innersourcing flow doc for Alaska airlines employees, contractors, and vendors

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Other (Document Update)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by jason-capsule42** [comment]
@Patrick-Daly-AA Can you apply the correct checkboxes in the PR body content for `Type of Change` and `Checklist`?

File: template/.github/CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Please take a moment to review this document in order to make the contribution process easy and effective for everyone involved. Also be sure to review the [Issues, pull requests and labels](https://auro.alaskaair.com/contributing/issues-prs-labels) process document.
 
+If you are an Alaska Airlines Employee, please see our [Innersourcing Flow Guide on Sharepoint](https://alaskaair.sharepoint.com/sites/ECOMt/Guilds/auro-users-and-contributors).
```
Comment (jason-capsule42) [inline template/.github/CONTRIBUTING.md:5]: Contractors would be able to see this too, correct?

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: template/.github/CONTRIBUTING.md (hunk 1 above)
Comment (Patrick-Daly-AA) [reply template/.github/CONTRIBUTING.md:5]: Good point.  let me make that more clear.

**Comment by Patrick-Daly-AA** [comment]
@jason-capsule42 , whoops, yup.  Got the check boxes and the PR body updated.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**